<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Categoria</class>
 <widget class="QDialog" name="Categoria">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Observação</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leCategoria">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>500</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSalvarEditar">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>400</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Salvar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbApagar">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>400</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Apagar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Categoria</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="pteObs">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>511</width>
     <height>87</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tbwCategoria">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>511</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
